src/Parser/Shaman/Restoration/Modules/Talents/HighTide.js

```diff
@@ -39,6 +45,24 @@ class HighTide extends Analyzer {
     // Store the id of the totem we summoned so that we don't include the EST of other rshamans.
     this.activeEST = event.targetID;
   }
+
+  statistic() {
+
+    return (
+      <StatisticBox
+        icon={<SpellIcon id={SPELLS.HIGH_TIDE_TALENT.id} />}
+        value={`${formatPercentage(this.healing)} %`}
+        label={(
+          <dfn data-tip={'The percentage of your healing that is caused by High Tide.'}>
```

Comment:
Tooltips like these would be best omitted to reduce the amount of tooltips so tooltips are primarily used for things that reveal interesting/important info so that users don't learn they are uninteresting.